src/xobject.rs

```diff
@@ -128,6 +128,16 @@ impl<'a> ImageXObject<'a> {
         self.insert(Name(b"Matte")).array().items(color);
         self
     }
+
+    /// Write the `/Metadata` attribute to specify the image's metadata. PDF
+    /// 1.4+.
+    ///
+    /// The reference shall point to a [metadata
+    /// stream](crate::writers::Metadata).
```

Comment:
either this can be written without `crate::writers::` or the link below is broken.